src/data/logic/redirection.ts

```diff
@@ -136,6 +145,16 @@ export const buildAvailableConfigurations = (version: number): Configuration[] =
         data: ntdLeishKenyaData,
         icon: "img/kenya.png",
     },
+    {
+        programme: "international-projects",
+        title: i18n.t("International Projects"),
+        description: i18n.t("Landing Page for International Projects"),
+        userGroupIds: [EFH_ADMIN, EFH_DASHBOARD, EFH_DATA_ENTRY, EFH_MIO, DATA_MONITORING],
+        page: InternationalLandingPage,
+        header: internationalHeader,
+        data: internationalData,
+        icon: "img/icon.png",
+    },
 ];
```

Comment:
This entry makes a redirect to the IP Landing page when some group is matched. But from the task description, we need something more: render the project monitoring app for DATA_MONITORING only-match and EFH app for the other groups only-match. Only if we belong to both groups we render the IP landing page. @adrian Note that "no group" scenario is not generically supported by the app; in that case, it renders the root landing page.

Reply:
@tokland i'm not entirely sure how to go about this. is this similar to what is done in the src/domain/models/hepatitis/Hepatitis.js file?

Reply:
We've discussed the approach with @adrianq, it's not trivial, he'll contact you on how to tackle this.

Reply:
okay. thanks @tokland